Programming/bundle/nlparser/src/main/webapp/jsp/tabs.jsp

```diff
@@ -34,6 +34,7 @@
 
                     <input id="search-input-${domainEntry.key}" name="q" type="text" class="input-large search-input"  
                            placeholder="Type your question.." data-searchdomain="${domainEntry.key}"></input>
+                    <input type="submit" style="position: absolute; left: -9999px"/>
```

Comment:
IE does not support display:none and visibility:hidden is also kind of broken, I think. See http://stackoverflow.com/questions/477691/submitting-a-form-by-pressing-enter-without-a-submit-button